h/formatters/interfaces.py

```diff
@@ -34,12 +34,12 @@ def preload(ids):  # noqa: N805
         :type ids: list of unicode
         """
 
-    def format(annotation):  # noqa: N805
+    def format(annotation_resource):  # noqa: N805
         """
         Presents additional annotation data that will be served to API clients.
 
-        :param annotation: The annotation object that needs presenting.
-        :type annotation: memex.models.Annotation
+        :param annotation_resource: The annotation that needs presenting.
+        :type annotation_resource: memex.resources.AnnotationResource
```

Comment:
I just merged #4495, so this (and all other references to `memex.resources.AnnotationResource`) in this PR need to be `h.resources.AnnotationResource`.